<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>821</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>241</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>250</green>
        <blue>243</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>241</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>250</green>
        <blue>243</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>241</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>250</green>
        <blue>243</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>250</green>
        <blue>243</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PAiNter</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>601</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>601</height>
     </size>
    </property>
    <widget class="QMdiArea" name="mdiArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>771</width>
       <height>601</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>771</width>
       <height>601</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="viewMode">
      <enum>QMdiArea::TabbedView</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="tabsMovable">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <widget class="QMenu" name="File">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>113</y>
      <width>145</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="Main">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actiontoDraw"/>
    <addaction name="actionTools"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Main"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actiontoDraw"/>
   <addaction name="actionTools"/>
  </widget>
  <widget class="QDockWidget" name="Tools">
   <property name="minimumSize">
    <size>
     <width>95</width>
     <height>599</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>599</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QComboBox" name="PenColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>black</string>
     </property>
     <item>
      <property name="text">
       <string>black</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/black.png</normaloff>:/images/images/black.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/blue.png</normaloff>:/images/images/blue.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>red</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/red.png</normaloff>:/images/images/red.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/green.png</normaloff>:/images/images/green.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>orange</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/orange.png</normaloff>:/images/images/orange.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>more</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/color.png</normaloff>:/images/images/color.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="PenWidth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/width1.png</normaloff>:/images/images/width1.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/width2.png</normaloff>:/images/images/width2.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/width3.png</normaloff>:/images/images/width3.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/Width4.png</normaloff>:/images/images/Width4.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/width5.png</normaloff>:/images/images/width5.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6pix</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/Width6.png</normaloff>:/images/images/Width6.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>PenColor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PenWidth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>27</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="CanvasColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>white</string>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtBottom</enum>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>white</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/white.png</normaloff>:/images/images/white.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yellow</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/yellow.png</normaloff>:/images/images/yellow.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/green.png</normaloff>:/images/images/green.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>red</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/red.png</normaloff>:/images/images/red.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>more</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/color.png</normaloff>:/images/images/color.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CanvasColor</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="Draw">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>89</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>66</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>66</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>toDraw</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>860</width>
      <height>43</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>43</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>901</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="Line">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Line</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/line.png</normaloff>:/images/images/line.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cut">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clip Line</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Ellipse">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ellipse</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/ellipse.png</normaloff>:/images/images/ellipse.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Polygon">
        <property name="text">
         <string>Polygon</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/polygon.png</normaloff>:/images/images/polygon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bezier">
        <property name="text">
         <string>Bezier Curve</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/curve.png</normaloff>:/images/images/curve.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bspline">
        <property name="text">
         <string>B-spline Curve</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/curve.png</normaloff>:/images/images/curve.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New(N)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(O)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(S)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/saveas.png</normaloff>:/images/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs(A)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiontoDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/toDraw.png</normaloff>:/images/images/toDraw.png</iconset>
   </property>
   <property name="text">
    <string>toDraw(D)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/Tools.png</normaloff>:/images/images/Tools.png</iconset>
   </property>
   <property name="text">
    <string>Tools(T)</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <zorder>Tools</zorder>
  <zorder>Draw</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
